pyhub/mcptools/google/cli_commands.py

```
@@ -0,0 +1,781 @@
+):
+    """Google API 인증 관리"""
+    # Ensure Google Sheets is enabled
+    os.environ.setdefault("USE_GOOGLE_SHEETS", "1")
+
+    auth = GoogleSheetsAuth()
+
+    if status:
+        # 인증 상태 확인
+        console.print("🔍 인증 상태 확인 중...")
+
+        if not auth.token_file.exists():
```

Pylint (3.3.7)

[error] Instance of 'GoogleSheetsAuth' has no 'token_file' member

(E1101)